engine/src/main/resources/templates/Common/insightsEmailBody.html

```diff
@@ -0,0 +1,520 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
We had many `insightsEmailBody.html` files until now. Which one did you use to build this V2 file?

Reply:
It is based on copy from policies.
If an application needs additionnal CSS, it could be done by adding it on section `extra-css` of its own template.